app.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const pokemon = require("./models/pokemon.json");
+const app = express();
+
+app.get("/", (req, res) => {
+    res.send("Intro to express response")
+})
+
+// project name generator
+
+app.get("/:verb/:adjective/:noun", (req, res) => {
+    const { verb, adjective, noun } = req.params;
+    res.send(`Congratulations on starting a new project called ${verb}-${adjective}-${noun}!`)
+})
+
+// bugs in the code
+
+app.get("/bugs", (req, res) => {
+    res.send(`
+        <p>99 little bugs in the code</p>
+        <p>99 little bugs</p>
+        <p><a href="/bugs/101">pull</a> one down, patch it around</p>
+        <p>101 little bugs in the code</p>
+    `)
+})
+
+app.get("/bugs/:numberOfBugs", (req, res) => {
+    const { numberOfBugs } = req.params;
+
+    if (numberOfBugs > 200) {
+        res.send(`
+    <p>${numberOfBugs} little bugs in the code</p>
+    <p>Too many little bugs, <a href="/bugs">start over</a></p>
+    `)
+    }
+    res.send(`
+    <p>${numberOfBugs} little bugs in the code</p>
+    <p>${numberOfBugs} little bugs</p>
+    <p><a href="/bugs/${Number(numberOfBugs) + 2}">pull</a> one down, patch it around</p>
+    <p>${Number(numberOfBugs) + 2} little bugs in the code</p>
+    `)
+})
+
+// poke-express
+
+app.get("/pokemon", (req, res) => {
+    res.send(`<ul>${pokemon.map(pokemon => `<li>${pokemon.name}</li>`)}</ul>`)
+})
+
+app.get("/pokemon/search", (req, res) => {
+    const { name } = req.query;
+
+    res.send(pokemon.find(pokemon => pokemon.name.toLowerCase() === name.toLowerCase()))
```

Comment:
This is going to send back `undefined` if the pokemon is not found. the direction you are heading in is correct but you want to handle the `.find()` outside of the `response`